src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1053,13 +1063,24 @@ private BidResponse toBidResponseWithCacheInfo(List<BidderResponse> bidderRespon
                 .build();
     }
 
+    private String getPublisherId(Site site, App app) {
```

Comment:
There is already a method `publisherId` for doing exactly the same. Also is it possible to pass `publisherId` that is calculated in `holdAuction` method?